components/PlanSubscriptionView.vue

```diff
@@ -52,21 +52,14 @@
                   class="inline"
                 >
                   <p class="font-light my-4">{{ plan.description }}</p>
-                  <ul role="list" class="mb-1 space-y-1 text-left" :class="checked ? 'text-white' : 'text-gray-900'">
+                  <ul
+                    v-for="entitlement in plan.entitlements"
+                    :key="entitlement.name"
+                    :value="entitlement.name"
+                    role="list" class="mb-1 space-y-1 text-left" :class="checked ? 'text-white' : 'text-gray-900'"
+                  >
                     <li class="flex items-center space-x-2">
-                        <span><span class="font-semibold">{{plan.projects}}</span> project</span>
-                    </li>
-                    <li class="flex items-center space-x-2">
-                        <span>Up to <span class="font-semibold">{{plan.clicks}}</span> clicks per month</span>
-                    </li>
-                    <li class="flex items-center space-x-2">
-                        <span>Supported project types: <span class="font-semibold"> {{ plan.project_types.join(', ') }}</span></span>
-                    </li>
-                    <li class="flex items-center space-x-2">
-                        <span :class="!plan.premium_button ? 'line-through' : ''">Premium project button</span>
-                    </li>
-                     <li class="flex items-center space-x-2">
-                        <span :class="!plan.analytics ? 'line-through' : ''">Analytics panel</span>
+                        <span v-html="entitlement.name" />
```

Comment:
I think it's fine, especially since we don't really force customers to do it like this, but we might wanna look at sanitizing this? Also, bear in mind, that this is generated by rendering a jinja template which is configured to autoescape.